code/modules/antagonists/cult/runes.dm

```diff
@@ -309,6 +309,29 @@ structure_check() searches for nearby cultist structures required for the invoca
 				to_chat(M, "<span class='cultlarge'>\"I accept this sacrifice.\"</span>")
 			else
 				to_chat(M, "<span class='cultlarge'>\"I accept this meager sacrifice.\"</span>")
+	
+	if(iscyborg(sacrificial))
+		var/static/list/constructs = list(
+			"Juggernaut" = image(icon = 'icons/mob/cult.dmi', icon_state = "juggernaut"),
+			"Wraith" = image(icon = 'icons/mob/cult.dmi', icon_state = "wraith"),
+			"Artificer" = image(icon = 'icons/mob/cult.dmi', icon_state = "artificer")
+			)
+		var/construct_class = show_radial_menu(first_invoker, sacrificial, constructs, require_near = TRUE, tooltips = TRUE)
+		if(QDELETED(sacrificial))
+			return FALSE
+		switch(construct_class)
+			if("Juggernaut")
+				makeNewConstruct(/mob/living/simple_animal/hostile/construct/juggernaut, sacrificial, first_invoker, FALSE, get_turf(src))
+			if("Wraith")
+				makeNewConstruct(/mob/living/simple_animal/hostile/construct/wraith, sacrificial, first_invoker, FALSE, get_turf(src))
+			if("Artificer")
+				makeNewConstruct(/mob/living/simple_animal/hostile/construct/artificer, sacrificial, first_invoker, FALSE, get_turf(src))
```

Comment:
On the same note, it might make sense to put this in some proc that shards can access as well.

The surface area of adding a new construct should be as minimal as possible.